Source/WebGPU/WGSL/AST/ASTStringDumper.cpp

```diff
@@ -265,6 +265,15 @@ void StringDumper::visit(UnaryExpression& expression)
     visit(expression.expression());
 }
 
+void StringDumper::visit(BinaryExpression& expression)
+{
+    constexpr ASCIILiteral binaryOperator[] = { "+"_s };
```

Comment:
Is there a way to enforce that these stay in the right order?

Reply:
There is, but I think we should eventually just use a switch case, but I mostly just followed what `UnaryExpression` is doing.